.changeset/giant-seas-complain.md

```diff
@@ -0,0 +1,38 @@
+---
+"@salt-ds/lab": minor
+---
+
+- Refactored `Drawer` to use floating-ui and Salt's `Scrim`.
+- Open prop set to false by default
```

Comment:
should all the changes be described in the same way? (refactored, added,fixed,set )